react-backend/client/src/components/home.js

```diff
@@ -1,11 +1,29 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
 
+const postFeedback = (cbId, feedbackScore) => {
+  const headers = new Headers({
+    Authorization: localStorage.getItem('token'),
+    'Content-Type': 'application/json',
+  });
+
+  return fetch('/api/cb/feedback', {
+    method: 'POST',
+    headers,
+    body: JSON.stringify({
+      cbId,
+      feedbackScore,
+    }),
+  });
```

Comment:
Then what? How does the visitor know that their feedback was successfully or unsuccessfully recorded?

Reply:
Are we gonna restrict how often feedback can be given? Some sort of waiting period, to stop people spamming. Maybe an issue for another time.

Reply:
☝️ Yeah, that was kinda my thinking. _Vaguely_ css related...

Reply:
Great shout @RogeredBacon. A frontend-only approach to abusive/malicious use cases is not a solution. Needs additional backend logic

Reply:
Oops, okay, I should have looked at the designs better, this is actually all on the wrong page and should have a redirect on the client... brb..